my_final_ project

emilyyaklich opened the pull request:
updating to the correct branch!

niezu committed:
my_final_ projecmy_final_ projectt

niezu commented:
I don't know how to do it.

emilyyaklich commented:
Hi! So, I was able to create the pull request myself (which is what this thread is) to the branch that is updated properly in your repository. I am going to submit a pull request to your repository soon. I will leave the comments here after I do that!

niezu commented:
Thank you.  I can't find the file in Jupiter notebook now.  I think we moved to the project_spring_2020 right? from ls, I only see project_spring_2020 right.
    On Thursday, May 7, 2020, 09:45:00 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Hi! So, I was able to create the pull request myself (which is what this thread is) to the branch that is updated properly. I am going to submit a pull request to your repository soon. I will leave the comments here after I do that!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

emilyyaklich commented:
Hi, 

I have just submitted a pull request to your niezu:master branch of your project_spring_2020 repository. In order to see this and merge the pull request with your branch, go to your *niezu/project_spring_2020* repository (make sure that you are on the **master** branch - you have another branch named patch-1 which does not contain the updated scripts) and click on the **pull requests** tab and then you can choose to **merge pull request** which will integrate my suggested updates into your repository. 

A little bit about the update suggestions that are included in the pull request:
- I have created a setup.py file (part of packaging your code) which you can fill out the information in this script like your name, email, short description, the name of the package etc. I have already added the URL to your repository in this file.
- In the *tests* directory of your repository I added an example test in the *sample_test.py* file that will test the 'test_convert_c_to_f' function from your script. You can add tests for your other functions to this same file as well. You can run your tests by using
```
pytest sample_test.py
```
in the terminal. 


- I moved your *my_final_project.ipynb* notebook into the *project_spring_2020/* directory in order to organize the repository. 

I think that covers most of the updates that I am suggesting. Another thing I would suggest is to make sure that the *my_final_project.ipynb* does not have any errors in the code itself. It might help to tidy up the notebook a bit so it only contains the definitions of specific functions that you use within the script. 

Let me know if you have any questions!

emilyyaklich commented:
and, yes, on your local computer the *my_final_project.ipynb* notebook should be located in the *project_spring_2020/* directory!

niezu commented:
Hi Thank you so much.  I really appreciate your help. I haven't look it yet, but I will do it soon.  I have another questions.1, How can save something I put in the read me?
2.  I found there are typo on the project.  How can I change it? the last formula should be /18, but it was -32.
Thank you.
    On Thursday, May 7, 2020, 11:21:26 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Hi,

I have just submitted a pull request to your niezu:master branch of your project_spring_2020 repository. In order to see this and merge the pull request with your branch, go to your niezu/project_spring_2020 repository (make sure that you are on the master branch - you have another branch named patch-1 which does not contain the updated scripts) and click on the pull requests tab and then you can choose to merge pull request which will integrate my updates into your repository.

A little bit about the update suggestions that are included in the pull request:
   
   - I have created a setup.py file (part of packaging your code) which you can fill out the information in this script like your name, email, short description, the name of the package etc. I have already added the URL to your repository in this file.
   - In the tests directory of your repository I added an example test in the sample_test.py file that will test the 'test_convert_c_to_f' function from your script. You can add tests for your other functions to this same file as well. You can run your tests by using
pytest sample_test.py

in the terminal.
   
   - I moved your my_final_project.ipynb notebook into the project_spring_2020/ directory in order to organize the repository.

I think that covers most of the updates that I am suggesting. Another thing I would suggest is to make sure that the my_final_project.ipynb does not have any errors in the code itself. It might help to tidy up the notebook a bit so it only contains the definitions of specific functions that you use within the script.

Let me know if you have any questions!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

emilyyaklich commented:
Did you edit your README.md file on github or from your local computer?

To fix the errors in your *my_final_project.ipynb* notebook you can open the notebook on your local computer in Jupyter Lab, make the edits and save them. You can then add and commit your changes using git and push them to your remote repository on github (this is using the **git add --all**, **git commit** and **git push** commands in your terminal.

niezu commented:
Hi, I know  how to save the changes now.
Nie
    On Thursday, May 7, 2020, 11:21:26 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Hi,

I have just submitted a pull request to your niezu:master branch of your project_spring_2020 repository. In order to see this and merge the pull request with your branch, go to your niezu/project_spring_2020 repository (make sure that you are on the master branch - you have another branch named patch-1 which does not contain the updated scripts) and click on the pull requests tab and then you can choose to merge pull request which will integrate my updates into your repository.

A little bit about the update suggestions that are included in the pull request:
   
   - I have created a setup.py file (part of packaging your code) which you can fill out the information in this script like your name, email, short description, the name of the package etc. I have already added the URL to your repository in this file.
   - In the tests directory of your repository I added an example test in the sample_test.py file that will test the 'test_convert_c_to_f' function from your script. You can add tests for your other functions to this same file as well. You can run your tests by using
pytest sample_test.py

in the terminal.
   
   - I moved your my_final_project.ipynb notebook into the project_spring_2020/ directory in order to organize the repository.

I think that covers most of the updates that I am suggesting. Another thing I would suggest is to make sure that the my_final_project.ipynb does not have any errors in the code itself. It might help to tidy up the notebook a bit so it only contains the definitions of specific functions that you use within the script.

Let me know if you have any questions!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

niezu commented:
I changed on GitHub and  edit the file and clicked commit changes.  Is it ok?
I received email from GitHub asked me to change the repository name, at that timeI hasn't received your e-mail.  I changed to Pathon-2020.  I don't know what will happen.  I look at it tomorrow.  Thank you for the help.

Nie
    On Thursday, May 7, 2020, 11:47:02 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Did you edit your README.md file from github or from your local computer?

To fix the errors in your my_final_project.ipynb notebook you can open the notebook on your local computer in Jupyter Lab, make the edits and save them. You can then add and commit your changes using git and push them to your remote repository on github (this is using the git add --all, git commit and git push commands in your terminal.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

niezu committed:
Update my_final_project.ipynb

niezu committed:
Update README.md

niezu committed:
changed at 05072020

niezu committed:
05072020-1

niezu committed:
05082020.md

niezu committed:
05082020.md

niezu committed:
005082020.md

niezu committed:
08052020.md

niezu commented:
Hi Emily,  I changer the project in my computer Jupiter lab.  In terminal , I typed git add --all (return), it doesn't work and I typed git add --all, git commit( return) It still doesn't work. It says that 'no git repository.
Nie  

    On Thursday, May 7, 2020, 11:47:02 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Did you edit your README.md file from github or from your local computer?

To fix the errors in your my_final_project.ipynb notebook you can open the notebook on your local computer in Jupyter Lab, make the edits and save them. You can then add and commit your changes using git and push them to your remote repository on github (this is using the git add --all, git commit and git push commands in your terminal.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

emilyyaklich commented:
Make sure that you are in your *project_spring_2020/* directory on your computer. In your terminal you can try typing: 
```
cd project_spring_2020
```
Then the git add --all, git commit, and git push commands should work.

niezu commented:
Ok, I will try, Thank you.
Nie

> On May 8, 2020, at 12:27 PM, emily yaklich <notifications@github.com> wrote:
> 
> 
> Make sure that you are in your project_spring_2020/ directory on your computer. In your terminal you can try typing:
> 
> cd project_spring_2020
> Then the git add --all, git commit, and git push commands should work.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/biof309/project_spring_2020/pull/31#issuecomment-625897670>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALJLHOT3ZTLTOUXPO23T5RTRQQXF5ANCNFSM4M3YU3OQ>.
>

niezu commented:
Hi ,
I did, after type the git commit ( return), It ask me enter the commit message for your changes.  I typed git push,  git push committed files.  I don’t know if it works.

Thank you.

> On May 8, 2020, at 12:27 PM, emily yaklich <notifications@github.com> wrote:
> 
> 
> Make sure that you are in your project_spring_2020/ directory on your computer. In your terminal you can try typing:
> 
> cd project_spring_2020
> Then the git add --all, git commit, and git push commands should work.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/biof309/project_spring_2020/pull/31#issuecomment-625897670>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALJLHOT3ZTLTOUXPO23T5RTRQQXF5ANCNFSM4M3YU3OQ>.
>

niezu commented:
Looks like it didn't go through because I checked the project in local computer.  The readme file should use the one I changed in my computerist is README 005082020.
Nie
    On Friday, May 8, 2020, 12:27:13 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Make sure that you are in your project_spring_2020/ directory on your computer. In your terminal you can try typing:
cd project_spring_2020

Then the git add --all, git commit, and git push commands should work.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

niezu commented:
Hi Emily, 
I am sorry to disturb you again.  I understand that the class is finished.  I followed your instruction and got the following result, and I don't know where is my final project file.  Could you help me with it?  Thank you very much.

(base) NCI-02181266-ML:project_spring_2020 niezu$ git add --all

(base) NCI-02181266-ML:project_spring_2020 niezu$ git commit

hint: Waiting for your editor to close the file... 

E325: ATTENTION

Found a swap file by the name "~/project_spring_2020/.git/.COMMIT_EDITMSG.swp"

          owned by: niezu   dated: Fri May  8 12:44:33 2020

         file name: ~niezu/project_spring_2020/.git/COMMIT_EDITMSG

          modified: YES

         user name: niezu   host name: NCI-02181266-ML

        process ID: 13222

While opening file "/Users/niezu/project_spring_2020/.git/COMMIT_EDITMSG"

             dated: Fri May  8 18:21:22 2020

      NEWER than swap file!




(1) Another program may be editing the same file.  If this is the case,

    be careful not to end up with two different instances of the same

    file when making changes.  Quit, or continue with caution.

(2) An edit session for this file crashed.

    If this is the case, use ":recover" or "vim -r /Users/niezu/project_spring_2

020/.git/COMMIT_EDITMSG"

    to recover the changes (see ":help recovery").

    If you did this already, delete the swap file "/Users/niezu/project_spring_2

020/.git/.COMMIT_EDITMSG.swp"

    to avoid this message.




Swap file "~/project_spring_2020/.git/.COMMIT_EDITMSG.swp" already exists!

-- More --


    On Friday, May 8, 2020, 12:27:13 PM EDT, emily yaklich <notifications@github.com> wrote:  
 
 


Make sure that you are in your project_spring_2020/ directory on your computer. In your terminal you can try typing:
cd project_spring_2020

Then the git add --all, git commit, and git push commands should work.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.

leesup commented:
Hello,

Could you try running the following in your project_spring_2020 folder?
git rm .git/.COMMIT_EDITMSG.swp
Then try running git commit again.

-Paul

niezu commented:
Hi Paul,

Thank you for the help,
I tried and I got the message as following:

E325: ATTENTION
Found a swap file by the name "~/project_spring_2020/.git/.COMMIT_EDITMSG.swp"
          owned by: niezu   dated: Fri May  8 12:44:33 2020
         file name: ~niezu/project_spring_2020/.git/COMMIT_EDITMSG
          modified: YES
         user name: niezu   host name: NCI-02181266-ML
        process ID: 13222
While opening file "/Users/niezu/project_spring_2020/.git/COMMIT_EDITMSG"
             dated: Wed May 13 00:10:57 2020
      NEWER than swap file!

(1) Another program may be editing the same file.  If this is the case,
    be careful not to end up with two different instances of the same
    file when making changes.  Quit, or continue with caution.
(2) An edit session for this file crashed.
    If this is the case, use ":recover" or "vim -r /Users/niezu/project_spring_2
020/.git/COMMIT_EDITMSG"
    to recover the changes (see ":help recovery").
    If you did this already, delete the swap file "/Users/niezu/project_spring_2
020/.git/.COMMIT_EDITMSG.swp"
    to avoid this message.

Swap file "~/project_spring_2020/.git/.COMMIT_EDITMSG.swp" already exists!
[O]pen Read-Only, (E)dit anyway, (R)ecover, (D)elete it, (Q)uit, (A)bort: 

> On May 11, 2020, at 3:07 AM, Suhwan Paul Lee <notifications@github.com> wrote:
> 
> 
> Hello,
> 
> Could you try running the following in your project_spring_2020 folder?
> git rm .git/.COMMIT_EDITMSG.swp
> Then try running git commit again.
> 
> -Paul
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/biof309/project_spring_2020/pull/31#issuecomment-626513854>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ALJLHOVWXEOASLXFXPQDFBDRQ6P2XANCNFSM4M3YU3OQ>.
>
